Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM registry.fedoraproject.org/fedora-minimal:37 AS builder
-
-RUN microdnf install -y systemd-devel golang git && microdnf clean all
+FROM golang:1.20 as builder
 
 WORKDIR /go/src/audito-maldito
 
 # pre-copy/cache go.mod for pre-downloading dependencies and only redownloading them in subsequent builds if they change
 COPY go.mod go.sum ./
 RUN go mod download && go mod verify
 
-COPY . .
+COPY cmd ./cmd
+COPY ingesters ./ingesters
+COPY internal ./internal
+COPY processors ./processors
+COPY main.go .
 
 RUN go build -o audito-maldito
 
 # Not using distroless nor scratch because we need the systemd shared libraries
-FROM registry.fedoraproject.org/fedora-minimal:37
-
+FROM alpine:3.17.3
 # NOTE(jaosorior): Yes, we need to be the root user for this case.
 # We need access to the journal's privileged log entries and the audit log in the future.
 USER 0
 
 COPY --from=builder /go/src/audito-maldito/audito-maldito /usr/bin/audito-maldito
 
-ENTRYPOINT [ "/usr/bin/audito-maldito" ]
+ENTRYPOINT [ "/usr/bin/audito-maldito" ]
```

Comment:
Given these changes, the journald reader will be broken by default. I suggest you add this to an alternative Dockerfile and have an alternate tag to ship this.

Reply:
With the namedpipe abstraction we are able to remove the journald ingester since the journald logic is handled by rsyslog. 

I was keeping in there in case there was a future use for journald ingester with audito-maldito. I ended up deleting the jouranld code. We can always pull it back in later if it is necessary.